nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard
# Vendors Update functionality

## Description
Adding update functionality for Vendors

## Related Issue
https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/issues/21

## Motivation and Context
This adds the update function for CRUD

## How Can This Be Tested?
Click the edit button and make changes to a Vendor.
The page should reload after clicking the Update Vendor button, showing all the vendors with the updates on the card that was edited.

## Types of changes
